gobblin-hive-registration/src/test/java/org/apache/gobblin/hive/policy/HiveSnapshotRegistrationPolicyTest.java

```diff
@@ -0,0 +1,54 @@
+package org.apache.gobblin.hive.policy;
```

Comment:
Welcome onboarding Nate ! 
You will need to add license header to each new class created. Check the wiki :https://cwiki.apache.org/confluence/display/GOBBLIN/Contributor%27s+Guide